docs/framework_config.rst

```diff
@@ -155,6 +155,22 @@ Here is an example of the :code:`async with` syntax:
             blah.append(new_blah)
         await ctx.send("The new blah value has been added!")
 
+There is also a :py:meth:`Group.all` method. This will return all the stored data associated
+with a specific config group as a :py:class:`dict`. By negating the need to excessively call config,
+this method can be particularly useful when multiple values are to be retrieved from the same group.
+
+Here is an example of :py:meth:`Group.all` usage:
+
+.. code-block:: python
+
+    @commands.command()
+    async def getall(self, ctx):
+        all_global_data = await self.config.all()
+        await ctx.send("Foobar is {foobar}, foo baz is {foo_baz}".format(
```

Comment:
Would `format_map` be better here? Or would that add a little extra confusion?

Reply:
> Would `format_map` be better here? Or would that add a little extra confusion?

I did consider that, but yes I had thought it would add more confusion as the main focus was the `all` method and not `str.format_map`, which some people may not be so familiar with.